### PR TITLE
Kraken: use configured endpoint for futures

### PR DESCRIPTION
**Pull request by geseq**
# PR Description

👋 

Another small fix. Kraken futures doesn't use the configured url right now. This PR fixes it.

Fixes # (issue)

## Type of change

Please delete options that are not relevant and add an `x` in `[]` as item is complete.

- [x] Bug fix (non-breaking change which fixes an issue)

## How has this been tested

- [x] go test ./... -race
- [x] golangci-lint run
- [x] Manually in verbose mode

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally and on Github Actions/AppVeyor with my changes

### CODE REVIEW

**Review by gloriousCode — CHANGES_REQUESTED** [review]
Thanks for your PR! I appreciate using the proper function, even if it doesn't change the url :call_me_hand:  Thanks for picking it up!

File: exchanges/kraken/kraken_futures.go [hunk 1]
```diff
@@ -316,9 +316,15 @@ func (k *Kraken) SendFuturesAuthRequest(ctx context.Context, method, path string
 			"Nonce":   nonce,
 		}
 
+		var url string
+		url, err = k.API.Endpoints.GetURL(exchange.RestFutures)
```
[the remark this answers is not in the document]
Comment (geseq) [reply exchanges/kraken/kraken_futures.go]: updated

**Review by gloriousCode — APPROVED** [review]
tACK! Thanks for that

File: exchanges/kraken/kraken_futures.go [hunk 2]
```diff
@@ -316,6 +316,12 @@ func (k *Kraken) SendFuturesAuthRequest(ctx context.Context, method, path string
 			"Nonce":   nonce,
 		}
 
+		var futuresURL string
+		futuresURL, err = k.API.Endpoints.GetURL(exchange.RestFutures)
```
Comment (shazbert) [inline exchanges/kraken/kraken_futures.go:320]: This is the same as the const in kraken.go, please change and then change the path variable name. 
image

**Review by shazbert — CHANGES_REQUESTED** [review]
One pedantic issue. soz.

File: exchanges/kraken/kraken_futures.go (hunk 2 above)
Comment (geseq) [reply exchanges/kraken/kraken_futures.go:320]: it looks like `krakenFuturesURL` is not being used anywhere, so I shuffled things around a bit.

Comment (shazbert) [reply exchanges/kraken/kraken_futures.go:320]: LGTM! Thanks heaps.

**Review by shazbert — APPROVED** [review]
tACK
```json
Kraken attempt 1 request path: https://futures.kraken.com/derivatives/api/v3/openorders
Kraken request header [Apikey]: [wowow]
Kraken request header [Authent]: [VXnI8g87nY0biVxG1yBGJUBwFWMnuCFyPsZrItu+feuUwTZPbC8J9z5Ne1AGkwtpih4x4PiZPrb3g1FeNig/+g==]
Kraken request header [Nonce]: [1667366043623110500]
Kraken request type: GET
```

**Review by gloriousCode — APPROVED** [review]

**Review by thrasher- — APPROVED** [review]
Nice, ACK!

**Pull request merged**